app/plugin_api_test.go

```diff
@@ -510,6 +509,22 @@ func TestPluginAPIRemoveTeamIcon(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestPluginAPIUpdateUserActive(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+	activeTrue, err := api.UpdateUserActive(th.BasicUser.Id, true)
```

Comment:
You will need to update the test.

Reply:
Sorry about that, updated the tests now.